WalkingChallenge/SettingsViewController.swift

```diff
@@ -0,0 +1,207 @@
+      make.left.equalTo(devicePicture.snp.right).offset(Style.Padding.p8)
+    }
+
+    deviceDescription.textColor = Style.Colors.grey
+    deviceDescription.snp.makeConstraints { (make) in
+      make.top.equalTo(deviceName.snp.bottom)
+      make.left.equalTo(deviceName.snp.left)
+    }
+  }
+}
+
+private enum TableSection: Int {
+  case device
+  case accountSettings
+  case teams
+  case helpAndSupport
+}
+
+struct SettingsInfo {
+  let title: String
+  let values: [SettingsValue]
+}
+
+struct SettingsValue {
+  let configValue: String
+}
+
+class SettingsSectionDataSource {
+  let pedometerSource = UserInfo.pedometerSource
+  // swiftlint:disable line_length
+  var settingsInfo: [SettingsInfo] = [
+    SettingsInfo(title: Strings.Settings.accountSettings,
+                  values: [
+                    SettingsValue.init(configValue: Strings.Settings.editProfile),
+                    SettingsValue.init(configValue: Strings.Settings.changeEmailAddress),
+                    SettingsValue.init(configValue: Strings.Settings.notificationsAndReminders)]),
+                    SettingsInfo(title: Strings.Settings.teams, values: [SettingsValue.init(configValue: Strings.Settings.changeTeams)]),
+                    SettingsInfo(title: Strings.Settings.helpAndSupport,
+                                 values: [SettingsValue.init(configValue: Strings.Settings.faq),
+                                          SettingsValue.init(configValue: Strings.Settings.contactUs)])
+    ]
+  // swiftlint:enable Line Length
+}
+
+fileprivate class SettingsDataSource: NSObject, UITableViewDataSource, UITableViewDelegate {
+  internal var dataSource: SettingsSectionDataSource?
+  override init() {
+    super.init()
+    self.dataSource = SettingsSectionDataSource()
+  }
+
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    if section == TableSection.device.rawValue {
+      return 1
+    } else {
+      let configuration = dataSource?.settingsInfo[section - 1]
+      return (configuration?.values.count)!
+    }
+  }
+
+  func numberOfSections(in tableView: UITableView) -> Int {
+    return (dataSource?.settingsInfo.count)!+1
+  }
+
+  func tableView(_ tableView: UITableView,
+                 cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    var cell: UITableViewCell
+
+    switch indexPath.section {
+    case TableSection.device.rawValue:
+      let pedometerSource = dataSource?.pedometerSource
+      cell = tableView.dequeueReusableCell(withIdentifier: PedometerCell.identifier, for: indexPath)
+      if let cell = cell as? PedometerCell {
+        cell.deviceName.text = pedometerSource?.rawValue
+        cell.deviceDescription.text = pedometerSource?.rawValue
+        cell.devicePicture.backgroundColor = UIColor.brown
+      }
+      break
+    default:
+      let configure = dataSource?.settingsInfo[safe: indexPath.section - 1]
+      let justValue = configure?.values[indexPath.row]
```

Comment:
Is it okay for configuration to be nil?  If not, change the `let` to a `guard let` perhaps?